### PR TITLE
feat(common): support gang scheduling config

### PR DESCRIPTION
**Pull request by shinytang6**
Try to resolve [issue 113](https://github.com/kubeflow/common/issues/113).

### CODE REVIEW

**Comment by shinytang6** [comment]
@googlebot I signed it!

**Review by terrytangyuan — COMMENTED** [review]
/assign @gaocegege @k82cn

File: pkg/apis/common/v1/types.go [hunk 1]
```diff
@@ -197,5 +197,8 @@ type RunPolicy struct {
 // SchedulingPolicy encapsulates various scheduling policies of the distributed training
 // job, for example `minAvailable` for gang-scheduling.
 type SchedulingPolicy struct {
-	MinAvailable *int32 `json:"minAvailable,omitempty"`
+	MinAvailable  *int32           `json:"minAvailable,omitempty"`
+	Queue         string           `json:"queue,omitempty"`
```
Comment (gaocegege) [inline pkg/apis/common/v1/types.go:201]: Should we make it a pointer?

File: pkg/controller.v1/common/job.go [hunk 2]
```diff
@@ -15,6 +15,7 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/runtime"
 	utilruntime "k8s.io/apimachinery/pkg/util/runtime"
+	"volcano.sh/volcano/pkg/apis/scheduling/v1beta1"
```
Comment (gaocegege) [inline pkg/controller.v1/common/job.go:18]: Is it ready to use?

@k82cn

File: pkg/apis/common/v1/types.go (hunk 1 above)
Comment (shinytang6) [reply pkg/apis/common/v1/types.go:201]: l just followed the spec of PodGroup in volcano. lf queue is empty, volcano will set it to "default" when scheduling.

**Comment by Jeffwan** [comment]
Overall it looks good to me

**Review by gaocegege — COMMENTED** [review]
/lgtm

**Comment by gaocegege** [comment]
@Jeffwan Should we release a version for common?

**Review by william-wang — APPROVED** [review]

**Comment by Jeffwan** [comment]
@gaocegege and I reviewed the codes. We can merge this one.  

/lgtm
/approve

**Comment by Jeffwan** [comment]
> @Jeffwan Should we release a version for common?

+1. This is user facing feature and we should cut a release.

**Pull request merged**